platform/linux-dpdk/pktio/dpdk.c

```diff
@@ -96,12 +96,12 @@ static void _dpdk_print_port_mac(uint8_t portid)
 		eth_addr.addr_bytes[5]);
 }
 
-static int input_queues_config_pkt_dpdk(pktio_entry_t *pktio_entry,
+static int input_queues_config_pkt_dpdk(odp_pktio_entry_t *pktio_entry,
 					const odp_pktin_queue_param_t *p)
 {
 	pktio_ops_dpdk_data_t *pkt_dpdk =
 		odp_ops_data(pktio_entry, dpdk);
-	odp_pktin_mode_t mode = pktio_entry->s.param.in_mode;
+	odp_pktin_mode_t mode = pktio_entry->param.in_mode;
```

Comment:
code 'union->s.member' compared to code 'struct->member':

they compiled into same assemble code, right?

and many odp data structure follow the union->s.member pattern, we may better keep it for consistency.

Reply:
You should not access pktio_entry_t from odp_pktio_ops_subsystem.h (since odp_pktio_ops_subsystem.h will become 'kind of' public API) but only a subset of it. This patch Is not necessary for performance.